### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by maksimpro001**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,67 @@ class Cargo:
...
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        super().__init__(name, weight)
+        self.coords = coords or [0, 0, 0]
```
Comment (ArsenPidhoretskyi) [inline app/main.py:41]:
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,66 @@ class Cargo:
...
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
```
Comment (EdAlekseiev) [inline app/main.py:11]: If an argument can be `None` you must show it in type hint.

Comment (EdAlekseiev) [inline app/main.py:11]:
```suggestion
            coords: list[int] | None = None
```

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,66 @@ class Cargo:
...
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
```
Comment (EdAlekseiev) [inline app/main.py:38]: The same comment as above

File: app/main.py [hunk 4]
```diff
@@ -2,4 +2,66 @@ class Cargo:
...
+    def go_down(self, step: int = 1) -> None:
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None,
```
Comment (EdAlekseiev) [inline app/main.py:54]: The same comment as above

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**